packages/mobile/package.json

```diff
@@ -60,7 +60,7 @@
     "@react-navigation/native": "^5.7.1",
     "@react-navigation/stack": "^5.7.1",
     "@segment/analytics-react-native": "^1.1.1-beta.2",
-    "@segment/analytics-react-native-firebase": "^1.1.1-beta.2",
+    "@segment/analytics-react-native-firebase": "1.1.1-beta.2",
```

Comment:
Why are we fixing this one and no the others?

Reply:
this was coming from a peer dependency resolution